tests/PHPStan/Rules/Arrays/data/nonexistent-offset-multi-dim.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+class X {}
```

Comment:
Typically these tests are named after the issue - so `bug-6605.php` and put in same namespace - `Bug6605` to ensure classes are unique.

Also, we shouldn't mix definitions and side effects, so ideally the code below `class X {}` should be as one of the methods in `X` instead.

Reply:
Ideally a separate file :)